packages/core/src/sass/app/_common/components/onfido-container.scss

```diff
@@ -22,8 +26,15 @@
     @include mobile() {
         padding: 1.6rem;
 
-        .onfido-sdk-ui-PageTitle-titleSpan {
-            font-size: 2rem;
+        & .onfido-sdk-ui-PageTitle {
+            &-titleWrapper {
```

Comment:
is camelcasing ok here?

Reply:
these are internal classes for onfido, we just make here some adjustments